deliverables/ruby_monk_problems/my_array.rb

```diff
@@ -6,6 +6,9 @@ def initialize(array)
   end
 
   def sum(initial_value = 0)
-    # your code here
+     return array.inject(:+) + initial_value unless block_given?
```

Comment:
Watch the whitespace.  Notice the first line of this method has one more space than the ones after it.